clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -275,6 +275,8 @@ def warn_drv_unknown_argument_clang_cl_with_suggestion : Warning<
   InGroup<UnknownArgument>;
 def err_drv_unknown_target_triple : Error<"unknown target triple '%0'">;
 
+def err_drv_stdlib_hardening_unavailable : Error<"libc++ hardening is available only when libc++ is used">;
```

Comment:
```suggestion
def err_drv_stdlib_hardening_unavailable : Error<"standard library hardening is available only when libc++ is used">;
```

Reply:
I disagree with that one change. As the option currently is only available for libc++ that should be reflected in the documentation. I agree that the internal names should be generic